fossilize.cpp

```diff
@@ -4701,10 +4908,12 @@ bool StateRecorder::Impl::copy_shader_module(const VkShaderModuleCreateInfo *cre
 bool StateRecorder::Impl::copy_sampler(const VkSamplerCreateInfo *create_info, ScratchAllocator &alloc,
                                        VkSamplerCreateInfo **out_create_info)
 {
-	if (create_info->pNext)
+	auto *info = copy(create_info, 1, alloc);
+
+	if (!copy_pnext_chain(info->pNext, alloc, &info->pNext))
```

Comment:
I think we're missing pNext hashing of samplers?

Reply:
Should be fixed now.